src/bundle/commitments.rs

```diff
@@ -28,10 +28,9 @@ fn hasher(personal: &[u8; 16]) -> State {
 /// personalized with ZCASH_ORCHARD_ACTIONS_HASH_PERSONALIZATION
 ///
 /// [zip244]: https://zips.z.cash/zip-0244
-pub fn hash_bundle_txid_data<'a, A: Authorization, V>(bundle: &'a Bundle<A, V>) -> Blake2bHash
-where
-    i64: From<&'a V>,
-{
+pub fn hash_bundle_txid_data<A: Authorization, V: Copy + Into<i64>>(
+    bundle: &Bundle<A, V>,
+) -> Blake2bHash {
```

Comment:
This change should go into the changelog, but I also don't see any usage of this outside of this crate (we use `Bundle::commitment` instead), so I'd argue we should actually remove it from the crate API. cc @nuttycom.

Reply:
Agreed, this seems like it could be crate-private.